modules/transcription-service-microsoft-azure/src/main/java/org/opencastproject/transcription/microsoftazure/MicrosoftAzureTranscriptionService.java

```diff
@@ -280,6 +287,24 @@ public void activate(ComponentContext cc) {
       logger.info("Default '{}' audio encoding will be used", compressedAudioFormat);
     }
 
+    // If text should be split into multiple cues
+    Option<String> isSplitTextOpt = OsgiUtil.getOptCfg(cc.getProperties(), SPLIT_TEXT);
+    if (isSplitTextOpt.isSome()) {
+      splitText = Boolean.valueOf(isSplitTextOpt.get());
+    }
+    logger.info("Long text will be split: {}", splitText);
+
+    // When splitting text into multiple cues, how many characters should each cue have at most
+    Option<String> splitTextLineSizeOpt = OsgiUtil.getOptCfg(cc.getProperties(), SPLIT_TEXT_LINE_SIZE);
+    if (splitTextLineSizeOpt.isSome()) {
+      try {
+        splitTextLineSize = Integer.parseInt(splitTextLineSizeOpt.get());
+      } catch (NumberFormatException e) {
+        throw new ConfigurationException("Invalid configuration for split text line size. Please check your"
+                + "configuration");
```

Comment:
But we are not using the default value, we're throwing an exception, right?

And throwing an exception make sense, since if the user went out of their way to modify the default they probably don't want the default, so we should notify them at the earliest opportunity?

Reply:
Derp, yeah you're right. This exception should also prevent the service from activating, so there's no default to use anyway ;D